apps/choleskey/tiled_cholesky_help.hpp

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include <cblas.h>
+#include <lapacke.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <cstdlib>
+#include <fstream>
+#include <sstream>
+#include "commons.hpp"
+
+double* generate_positiveDefinitionMatrix(const size_t matrix_size) {
+    double* A_matrix = new double[matrix_size * matrix_size];
```

Comment:
would recommend defining `data_type` in the header file and using that instead of double/int for portability.